godtools/App/Share/SwiftUI Views/BannerView/BannerViewModel.swift

```diff
@@ -0,0 +1,30 @@
+//
+//  BannerViewModel.swift
+//  godtools
+//
+//  Created by Rachael Skeath on 6/27/22.
+//  Copyright © 2022 Cru. All rights reserved.
+//
+
+import Foundation
+
+protocol BannerViewModelDelegate: AnyObject {
+    func closeBanner()
+}
+
+class BannerViewModel: NSObject, ObservableObject {
```

Comment:
Hey @levieggertcru, yeah I went back and forth on this-- my goal was to not have to duplicate the `closeBanner()` and `BannerViewModelDelegate` logic, since most views that include the BannerView will need a delegate in their viewModel to close it.  But at the same time, requiring a BannerViewModel on the BannerView itself seemed like it could be unnecessary in cases where we don't need the delegate, then it makes sense to just use the closeButtonTapHandler.  So this was my attempt at compromising.  Would you still rather I just remove it and let the views that use BannerView deal with closing it?  Or keep it, and require a BannerViewModel in BannerView? (does that make sense, that's a lot of words lol)

Reply:
Hey @rachaelblue I see what you're saying.  If BannerViewModel had some shared logic in closeTapped() between all of the banner views then I could see a use for it.  I'd say for now just remove.

Reply:
Fixed.